tropestogo/service/crawler/crawler.go

```diff
@@ -22,24 +22,31 @@ const (
 	acceptLanguageHeader          = "es"
 	upgradeInsecureRequestsHeader = "1"
 
+	// TvTropes date formats
+	tvTropesHistoryDateFormat = "Jan 2 2006 at 3:04:05 PM"
+
 	// Referer header for a well-known and trusty webpage
 	refererHeader = "https://www.google.com/"
 
-	TvTropesHostname       = "tvtropes.org"
-	TvTropesWeb            = "https://" + TvTropesHostname
-	TvTropesPmwiki         = TvTropesWeb + "/pmwiki/"
-	WorkPageSelector       = "table a"
-	CurrentSubpageSelector = ".curr-subpage"
-	SubWikiSelector        = "a.subpage-link:not(" + CurrentSubpageSelector + ")"
-	SubPageSelector        = "ul a.twikilink"
-	PaginationNavSelector  = "nav.pagination-box > a"
+	TvTropesHostname        = "tvtropes.org"
+	TvTropesWeb             = "https://" + TvTropesHostname
+	TvTropesPmwiki          = TvTropesWeb + "/pmwiki/"
+	WorkPageSelector        = "table a"
+	CurrentSubpageSelector  = ".curr-subpage"
+	SubWikiSelector         = "a.subpage-link:not(" + CurrentSubpageSelector + ")"
+	SubPageSelector         = "ul a.twikilink"
+	PaginationNavSelector   = "nav.pagination-box > a"
+	WorkHistoryPageSelector = "li.link-history a"
+	LastUpdatedSelector     = "#main-article > div:first-of-type .pull-right a"
 )
 
 var (
-	ErrNotFound = errors.New("couldn't request the URL")
-	ErrCrawling = errors.New("there was an error crawling TvTropes")
-	ErrEndIndex = errors.New("there's no next page on the index")
-	ErrParse    = errors.New("couldn't parse the HTML contents of the page")
+	ErrNotFound    = errors.New("couldn't request the URL")
```

Comment:
You should have used refactoring to convert this into real errors. Again, it's not useful to know that an URL can't be requested (whatever that means) if you don't know what's the actual URL.

Reply:
La url la añado luego al formatear el error al devolverlo, no se como formar un objeto de tipo "error" donde parte del texto sea variable que no sea esa forma. Investigare de todos modos otras formas de llevarlo